tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/extensions/server/BallerinaASTRequest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Incorrect year. This should be the year which the code gets merged. Some other files need to be updated as well.

Reply:
Fixed here and other related places as well.